.circleci/config.yml

```diff
@@ -49,6 +58,23 @@ workflows:
             parameters:
               platform: [linux, macos]
               rust_channel: [stable]
+  release:
+    jobs:
+      - test:
+          name: Test (<< matrix.rust_channel >> rust on << matrix.platform >>)
+          matrix:
+            parameters:
+              platform: [linux]
+              rust_channel: [stable]
+          <<: *crate_release
+      - publish:
```

Comment:
We'll want a `requires` key on the `publish` job to depend on the `test` jobs, [similar to federation-rs](https://github.com/apollographql/federation-rs/blob/0757a29c9fb58b2c7f1c9e5ffd9f4eb2a88c6acd/.circleci/config.yml#L101).

Reply:
was wondering whether we should just collapse it to a single thing (so we wouldn't have to setup env twice). might do this later